L1Trigger/TrackTrigger/plugins/TTStubBuilder.cc

```diff
@@ -158,10 +158,10 @@ void TTStubBuilder<Ref_Phase2TrackerDigi_>::produce(edm::Event& iEvent, const ed
 
         tempTTStub2.addClusterRef((tempTTStub.getClusterRef(0)));
         tempTTStub2.addClusterRef((tempTTStub.getClusterRef(1)));
-        tempTTStub2.setTriggerDisplacement(2. * tempTTStub.getTriggerDisplacement());
-        tempTTStub2.setTriggerOffset(2. * tempTTStub.getTriggerOffset());
-        tempTTStub2.setRealTriggerOffset(2. * tempTTStub.getRealTriggerOffset());
-        tempTTStub2.setHardwareBend(tempTTStub.getHardwareBend());
+        tempTTStub2.setRawBend(2. * tempTTStub.getRawBend());
```

Comment:
maybe worth specifying conversions between full and half units as named constants, to make the operations clearer